src/cockpit/samples.py

```diff
@@ -398,7 +398,15 @@ def sample(self, samples: Samples) -> None:
                 if line[0] != '/':
                     continue
 
-                path = line.split()[1]
+                split_line = line.split()
+                # Skip read-only loop mounts
+                if '/loop' in split_line[0] and 'ro' in split_line[3].split(','):
+                    continue
+                # Hide flatpaks
+                if 'revokefs-fuse' in split_line[0] and 'flatpak' in split_line[1]:
+                    continue
```

Comment:
What about:

```
tmpfs /run/credentials/systemd-journald.service tmpfs ro,nosuid,nodev,noexec,relatime,nosymfollow,size=1024k
```

Mount point exists, but we likely don't want it.

Reply:
Won't this get skipped by the:

```
# Only look at real devices
if line[0] != '/':
    continue
```

Reply:
Yes :)